pydis_site/apps/home/tests/test_views.py

```diff
@@ -7,3 +7,17 @@ def test_index_returns_200(self):
         url = reverse('home')
         resp = self.client.get(url)
         self.assertEqual(resp.status_code, 200)
+
+
+class TestLoginCancelledReturns302(TestCase):
+    def test_login_cancelled_returns_302(self):
```

Comment:
can we docstring these? It's nice because when the tests are run, it will display the docstrings instead of the method names as it lists them, and that looks nice. I know the function name is self-documenting, but I'd rather read a natural language sentence that describes the same thing.

No need for a long docstring here, just a one-liner is fine.

same with the method below.